tests/unit/output/test_local.py

```diff
@@ -48,6 +49,16 @@ def test_str_on_absolute_path(dvc):
     path = os.path.abspath(os.path.join("path", "to", "file"))
     output = OutputLOCAL(stage, path, cache=False)
 
+    assert path != str(output)
+    assert path == os.path.abspath(str(output))
```

Comment:
It is better to write this test in a positive form to check what it should be rather than what it shouldn't be:
```
assert output.def_path == os.path.join("path", "to", "file")
assert fspath(output.path_info) == path
```